src/runtime-rs/README.md

```diff
@@ -120,6 +120,24 @@ See the
 See the
 [debugging section of the developer guide](../../docs/Developer-Guide.md#troubleshoot-kata-containers).
 
+For diagnosing runtime configuration, use the `kata-ctl env` command:
+
+```bash
+# Display the currently applied Kata configuration in JSON format
+kata-ctl env --json
+
+# Display configuration details in TOML format
```

Comment:
It support two formats TOML and JSON, and here, the TOML format is the default format, right ? please add document for it.